converter/dicom-cast/src/Microsoft.Health.DicomCast.Core.UnitTests/Features/Fhir/FhirTransactionExecutorTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// -------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------
```

Comment:
why is it flagging this as changed

Reply:
File encoding based on our `.gitattributes` file